_getting-started/index.md

```diff
@@ -85,6 +85,8 @@ Run the following command in your terminal, following the on-screen instructions
 {% endtabs %}
 <!-- End tabs -->
 
+You might need to restart your terminal/shell after you install the tool using the package manager mentioned above.
```

Comment:
I had added a comment similar to this in the "testing your setup" drop down, I suggest to merge the two at this line, and perhaps surround with a block quote?